tests/bug858.js

```diff
@@ -0,0 +1,10 @@
+import {assert} from "./assert.js";
+
+Error.stackTraceLimit = Infinity;
+assert(new Error("error").stack.includes("bug858.js")); // stack should not be empty
+
+Error.stackTraceLimit = -Infinity;
+assert(new Error("error").stack.includes("bug858.js")); // stack should not be empty
```

Comment:
This and next are incorrect. There must be no stack trace. That's how V8 behaves. Have you observed otherwise?